### PR TITLE
importing models from _client

### PR DESCRIPTION
**Pull request by adelebai**
Expose some client models for the user (e.g. JobStatus)

### CODE REVIEW

File: azure-quantum/azure/quantum/__init__.py [hunk 1]
```diff
@@ -8,5 +8,7 @@
 from .job import *
 from .workspace import *
 
+from ._client.models import *
```
Comment (guenp) [inline azure-quantum/azure/quantum/__init__.py:11]:
```suggestion
from ._client.models._quantum_client_enums import JobStatus
```

**Review by guenp — APPROVED** [review]
I would suggest to use a direct import from `._client.models._quantum_client_enums` instead of the wildcard `from foo import *`; the latter also makes other models available that we want to keep private.

File: azure-quantum/azure/quantum/__init__.py (hunk 1 above)
Comment (adelebai) [reply azure-quantum/azure/quantum/__init__.py:11]: I think it's ._client.models import JobStatus (there's no _quantum_client_enums module). 
Let me make the new suggestion

Comment (guenp) [reply azure-quantum/azure/quantum/__init__.py]: Yes, it is also available via _client.models, but the class is defined here: https://github.com/microsoft/qdk-python/blob/main/azure-quantum/azure/quantum/_client/models/_quantum_client_enums.py#L36

Comment (adelebai) [reply azure-quantum/azure/quantum/__init__.py]: ok changed

**Pull request merged**